calendar_af_360/i18n/sv_SE.po

```diff
@@ -4,27 +4,58 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 12.0-20200508\n"
+"Project-Id-Version: Odoo Server 12.0-20200603\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-10-06 10:25+0000\n"
-"PO-Revision-Date: 2020-10-06 12:27+0200\n"
+"POT-Creation-Date: 2020-11-19 15:38+0000\n"
+"PO-Revision-Date: 2020-11-19 16:40+0100\n"
 "Last-Translator: <>\n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: \n"
-"Language: sv_SE\n"
+"Language: sv\n"
```

Comment:
Is this change correct?

Reply:
That's how the file is exported by my machine, it works in hr_office so it should work here too.